README.md

```diff
@@ -11,10 +11,13 @@ Please follow the instructions of that repo which is also written below.
 
 ===================================================
 
+In permutohedral_lattice/CMakeLists.txt, set the path to the **libtensorflow_framework.so** file in line 8.
```

Comment:
A brief description or example of where this file is would be great.

Reply:
Gotcha. Will add that.